File: 75. Sort Colors.c
void sortColors(int* nums, int numsSize){
    int r=0,w=0,b=0;
    for (int i = 0; i < numsSize; i++)
    {
        if(nums[i]==0){
            r++;
        }
        if(nums[i]==1){
            w++;
        }
        if(nums[i]==2){
            b++;
        }
    }
    for(int i=0;i<r;i++){
        if(i==numsSize){
            break;
        }
        nums[i]=0;
    }
    for(int i=0;i<w;i++){
        if(r==numsSize){
            break;
        }
        nums[r]=1;
        r++;
    }
    for(int i=0;i<b;i++){
        if(r==numsSize){
            break;
        }
        nums[r]=2;
        r++;
    }
}

File: 75. Sort Colors.py
class Solution:
    def sortColors(self, nums: List[int]) -> None:
        """
        Do not return anything, modify nums in-place instead.
        """
        red=nums.count(0)
        white=nums.count(1)
        blue=nums.count(2)
        for i in range(len(nums)):
            if red:
                nums[i]=0
                red-=1
            elif white:
                nums[i]=1
                white-=1
            else:
                nums[i]=2
                blue-=1
